www/docusaurus.config.js

```diff
@@ -1,8 +1,6 @@
-// @ts-check
-/* eslint-disable @typescript-eslint/no-var-requires */
-
-const { parseEnv } = require('./src/utils/env');
-const { generateTypedocDocusaurusPlugins } = require('./docusaurus.typedoc.js');
+// import type { Config } from '@docusaurus/types';
```

Comment:
Please get the `// @ts-check` back in
```suggestion
// @ts-check
```